qiskit/opflow/converters/circuit_sampler.py

```diff
@@ -334,6 +387,34 @@ def sample_circuits(
             ready_circs, had_transpiled=self._transpile_before_bind
         )
 
+        # restore the original parameter table and global phase in shadow_circs
+        for i, circ in enumerate(self._reuse_circs):
+            for j, _ in enumerate(circ):
+                for param in self._reuse_parameter_tables[i][j]:
```

Comment:
`self._reuse_parameter_tables[i][j].items()` is readable since `self._reuse_parameter_tables[i][j][param]` is frequently appeared.